commands/registrySettings.ts

```diff
@@ -1,36 +1,40 @@
+import * as assert from 'assert';
 import * as vscode from 'vscode';
 import { AzureRegistryNode } from "../explorer/models/azureRegistryNodes";
 import { CustomRegistryNode } from "../explorer/models/customRegistryNodes";
 import { DockerHubOrgNode } from "../explorer/models/dockerHubNodes";
 
+const defaultRegistryPathKey = "defaultRegistryPath";
+const defaultRegistryKey = "defaultRegistry";
+const combineRegistryPathsKey = "combineRegistryPathsFlag"
```

Comment:
This name means absolutely nothing.  How about 'hasCheckedRegistryPaths' (and reverse its boolean meaning)?